src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -86,7 +86,7 @@ internal class TextBufferIssueTracker : IIssueTracker, ITagger<IErrorTag>, IDisp
             this.FilePath = document.FilePath;
             this.charset = document.Encoding.WebName;
 
-            this.Factory = new IssuesSnapshotFactory(new IssuesSnapshot(GetProjectName(), GetProjectGuid(), FilePath, new List<IAnalysisIssueVisualization>()));
+            this.Factory = new IssuesSnapshotFactory(EmptyIssuesSnapshot.Create(FilePath));
```

Comment:
@rita-sonarsource as discussed offline, I'll remove this parameter from the factory and make the class nested internal.